packages/appstore/src/components/wallets-banner/__tests__/wallets-banner.spec.tsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
```

Comment:
Please, delete `waitFor`, you don't use it

Reply:
okay 😭